SofascoreAcademy/SofascoreAcademy/Styling.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Styling.swift
+//  SofascoreAcademy
+//
+//  Created by Akademija on 18.03.2024..
+//
+
+import Foundation
+import UIKit
+
+public var RobotoBold = UIFont(name: "Roboto-Bold", size: 14)
+public var RobotoRegular14 = UIFont(name: "Roboto-Regular", size: 14)
+public var RobotoRegular12 = UIFont(name: "Roboto-Regular", size: 12)
+public var RobotoCondensedRegularMicro = UIFont(name: "RobotoCondensed-Regular", size: 12)
+
+public var surfaceLv2 = UIColor(red: 0, green: 0, blue: 0, alpha: 0.4)
+public var red = UIColor.red
+public var surfaceLv4 = UIColor(white: 18.0 / 255.0, alpha: 0.1)
```

Comment:
```suggestion
public enum Styling {
 static let RobotoBold = UIFont(name: "Roboto-Bold", size: 14)
 }
```

Samo primjer da mozes dodati sve u enum Styling to ako zelis, ali bi bilo dobro da sve promjenis u let jer su to konstante, to je nesto sto se nece mijenjati